mono/metadata/assembly.c

```diff
@@ -89,7 +89,7 @@ static char **assemblies_path = NULL;
 /* Contains the list of directories that point to auxiliary GACs */
 static char **extra_gac_paths = NULL;
 
-#ifndef DISABLE_ASSEMBLY_REMAPPING
+#ifndef DISABLE_DESKTOP_LOADER
```

Comment:
Perhaps.  I was trying to stay consistent with all the other `--enable-minimal` flags which all define `DISABLE_feature` if that particular subsystem is excluded.  I thought that was the idiom in the runtime.